### PR TITLE
added login and sign up page

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

**Comment by swstica** [comment]
Please review these two previews:
https://deploy-preview-2--trusting-hopper-fc9917.netlify.app/signup-login/signup.html
https://deploy-preview-2--trusting-hopper-fc9917.netlify.app/signup-login/login.html

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <header>
         <div class="navbar">
             <div class="nav-items flex-flow-row">
-                <div class="nav-item"><a><img src="#">Thrift store</a></div>
+                <div class="nav-item"><a><img src="images/logo.png" height="40rem" width="100rem"></a></div>
```
Comment (pradipgharge) [inline index.html:16]: Avoid inline styling here.

File: signup-login/login.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            <div class="input-container flex-flow-column">
+                <img src="../images/logo.png" height="70px" width="70%">
+                <div class="input-box">
+                    <label>Email or mobile no.<span>*</span></label>
+                    <input type="text" class="input" placeholder="Username">                      
+                </div>
+                <div class="input-box password">
+                    <label>Password<span>*</span></label>
+                    <input type="password" class="input" placeholder="Password">
+                    <a href="#">Forget password?</a>
```
Comment (pradipgharge) [inline signup-login/login.html]: It should be forgot & not forget !

**Review by pradipgharge — COMMENTED** [review]
Suggested some minor changes, Also you can increase the font size in form inputs looks to small currently.

File: signup-login/login.html (hunk 2 above)
Comment (swstica) [reply signup-login/login.html]: corrected

File: index.html (hunk 1 above)
Comment (swstica) [reply index.html:16]: Actually, it doesn't work similarly in external styling. This attribute (like alt), not styling.

File: signup-login/login.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                  </div>
+            </div>
+        </div>
+
+    </header>
+
+    <section>
+        <div class="login-container">
+            <div class="input-container flex-flow-column">
+                <img src="../images/logo.png" height="70px" width="70%">
```
Comment (nikhiljugale007) [inline signup-login/login.html]: you should give alt for images

File: signup-login/login.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                    <label>Password<span>*</span></label>
+                    <input type="password" class="input" placeholder="Password">
+                    <a href="#">Forgot password?</a>
+                </div>
+                <div class="input-box">
+                    <span id="password-validation"></span>
+                    <button class="input-btn" onclick="validation()">Login</button>
+                    <script>
+                            let validation = () => document.getElementById("password-validation").innerHTML = "Wrong password, try again!"; 
+                    </script>
```
Comment (nikhiljugale007) [inline signup-login/login.html:54]: You can write this script in other js files else at the top or bottom.

File: signup-login/signup.css [hunk 5]
```diff
@@ -0,0 +1,38 @@
+@import "../index.css";
+@import url("https://radiantui.netlify.app/input/input.css");
+body{
+    background-color: #fff3f0;
```
Comment (nikhiljugale007) [inline signup-login/signup.css]: use css variables for colors

**Review by nikhiljugale007 — COMMENTED** [review]
Can add padding around forgot password. 
Suggested some changes in code.

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks good  to me .

File: index.html (hunk 1 above)
Comment (Sahil25061999) [reply index.html:16]: I dont think inline styling takes rem as unit, I am not sure though. I tried adding height: 40px and width: 100px  and it worked, so probably you can avoid inline style here.

**Review by Sahil25061999 — APPROVED** [review]
In terms of ui  I think you can add a bit of margin between and the label and input container, also increase the size of label, rest looks good to me.

File: signup-login/signup.css (hunk 5 above)
Comment (swstica) [reply signup-login/signup.css]: done

File: signup-login/login.html (hunk 3 above)
Comment (swstica) [reply signup-login/login.html]: done

File: signup-login/login.html (hunk 4 above)
Comment (swstica) [reply signup-login/login.html:54]: done

**Pull request merged**